fatal error occurred spawning output processexeception info:'FileNotFoundError' exception in errorwindow.py
Traceback (most recent call last):
  File "/Users/lisa/X/lisabu/cellanneal_gui_dev/cellanneal/cellanneal-gui/errorwindow.py", line 187, in __getattr__
    stderr=subprocess.PIPE).stdin
  File "/Users/lisa/opt/anaconda3/envs/cellanneal/lib/python3.7/subprocess.py", line 800, in __init__
    restore_signals, start_new_session)
  File "/Users/lisa/opt/anaconda3/envs/cellanneal/lib/python3.7/subprocess.py", line 1551, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '"/Users/lisa/opt/anaconda3/envs/cellanneal/bin/python" "/Users/lisa/X/lisabu/cellanneal_gui_dev/cellanneal/cellanneal-gui/errorwindow.py" "cellgui.py" "stderr"': '"/Users/lisa/opt/anaconda3/envs/cellanneal/bin/python" "/Users/lisa/X/lisabu/cellanneal_gui_dev/cellanneal/cellanneal-gui/errorwindow.py" "cellgui.py" "stderr"'
